CMakeLists.txt

```diff
@@ -33,5 +37,20 @@ set(CMAKE_CUDA_EXTENSIONS OFF)
 # With CUDA language enabled above, this should find the toolkit alongside the compiler
 find_package(CUDAToolkit REQUIRED)
 
+#Find VecCore with correct backend
+set(VecCore_VERSION 0.5.2)
+set(VecCore_BACKEND CUDA)
+find_package(VecCore ${VecCore_VERSION} REQUIRED COMPONENTS ${VecCore_BACKEND})
+message(STATUS "Using VecCore version ${VecCore_VERSION}")
+set(ADEPT_LIBRARIES_EXTERNAL ${ADEPT_LIBRARIES_EXTERNAL} VecCore::VecCore)
+
+# Find VecGeom geometry headers library
+find_package(VecGeom REQUIRED)
```

Comment:
Should we set a minimum VecGeom version as for VecCore? Could we use the latest 1.1.7 release?